### ISSUE TITLE
Add filter for letters queryset in campaign

### ISSUE THREAD

**Issue by petrikoz** [ITCase-django/django-unisender#60]
On page "Add campaign" I can select letters which was sent. But when I save form I have error **AK091110-08 [List #4031295 of letter #20144755 is deleted]**.

May be add filter for queryset of sended letters?
##

**Comment by mr-africa** [comment]
You can resending messages. 

But in your error message saying about you try to send already deleted letter

**Comment by petrikoz** [comment]
I try send letter with alredy deleted subscribe list. But sended letter I
can't edit again.

2014-08-26 15:26 GMT+06:00 Alexey Efimov notifications@github.com:

> You can resending messages.
> 
> But in your error message saying about you try to send already deleted
> letter
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/ITCase/django-unisender/issues/60#issuecomment-53395754
> .

## 

С уважением, Пётр Олегович Зеленин.

**Comment by mr-africa** [comment]
> But sended letter I can't edit again. 

yeah you can't editing email message  see https://github.com/ITCase/django-unisender/issues/61

**Comment by petrikoz** [comment]
This is wrong:
1. I can delete subscribe list which tied to some letters.
2. I can selected letters with deleted subscribe list in campaign. Because
   it not say me what it corrupted.
3. I get error!

2014-08-26 16:40 GMT+06:00 Alexey Efimov notifications@github.com:

> But sended letter I can't edit again.
> 
> yeah you can't editing email message see #61
> https://github.com/ITCase/django-unisender/issues/61
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/ITCase/django-unisender/issues/60#issuecomment-53402537
> .

## 

С уважением, Пётр Олегович Зеленин.
